### DISCUSSION TITLE
Can not render strong if brackets contains between `**`

### DISCUSSION THREAD

**Discussion by kIl3rr** [hexojs/hexo#4852] [Question / Answer] [answered]
![微信截图_20220103212945](https://user-images.githubusercontent.com/74661756/147936142-551c1441-99f9-4846-8122-305545ccafa8.png)
![微信截图_20220103213033](https://user-images.githubusercontent.com/74661756/147936148-393c5fec-e2c3-49a1-a51b-0d5610d45d41.png)

**Comment by yoshinorin** [comment, marked as answer]
@wo02ie 

* Which version of hexo are you using?
* Which markdown-renderer are you using? (e.g, hexo-renderer-marked
* Which theme are you using?
* Please see raw markdown file

**Reply by kIl3rr** [reply]
hexo: 6.0.0
hexo-cli: 4.3.0
theme： fluid
hexo-generator-archive@1.0.0
hexo-generator-category@1.0.0
 hexo-generator-index@2.0.0
hexo-renderer-marked@4.1.0



use * wrap syntax :

![图片](https://user-images.githubusercontent.com/74661756/148051899-85caa6f1-f1fb-47e7-8153-548541dd5344.png)



Is there anything else I need to provide😀

**Reply by yoshinorin** [reply]
@wo02ie 
It caused by [markedjs](https://github.com/markedjs/marked).
Please add space after `**` like belows.

```diff
- **組(group)**是用
+ **組(group)** 是用
```

It will be works well :)

![markdown](https://user-images.githubusercontent.com/11273093/148054296-923c847c-a2d3-4faa-84b2-54d797b6d537.jpg)

> [live demo](https://marked.js.org/demo/?text=**hoo(bar)**aaa%0A%0A**hoo(bar)**%20aaa&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22extensions%22%3A%20null%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=3.0.8)

**Reply by kIl3rr** [reply]
well,it works
thank u :p
